source/popup.js

```diff
@@ -17,7 +17,7 @@ document.addEventListener('DOMContentLoaded', function(){
 
   toggleButton.onclick = function(element) {
     let input = document.getElementById('mtgGuildColorsInput');
-    let currentState = input.value == 'true' || input.value == true ? true : false;
```

Comment:
`input.value === true ? true : false` is identical to `input.value === true`, so I simplified it.